website/docs/r/autoscaling_policy.html.markdown

```diff
@@ -184,6 +187,89 @@ The following arguments are supported:
 * `predefined_metric_type` - (Required) Describes a scaling metric for a predictive scaling policy. Valid values are `ASGAverageCPUUtilization`, `ASGAverageNetworkIn`, `ASGAverageNetworkOut`, or `ALBRequestCountPerTarget`.
 * `resource_label` - (Required) A label that uniquely identifies a specific Application Load Balancer target group from which to determine the request count served by your Auto Scaling group.
 
+##### customized_scaling_metric_specification
+The following arguments are supported:
+
+* `metric_data_queries` - (Required) A list of up to 10 structure that defines custom scaling metric in predictive scaling policy
```

Comment:
Thank you for you advice.
Actually, I have a concern here regarding the naming of `metric_data_queries`.
I AWS [CLI](https://docs.aws.amazon.com/autoscaling/ec2/APIReference/API_PredictiveScalingCustomizedCapacityMetric.html) the parameter here is a list of `metric_data_query` objects where the list is called `metric_data_queries`, that is to say, the config will look like this is AWS CLI:
```
{
 ...
 customized_scaling_metric_specification: {
   metric_data_queries: [
     {
         id = "xxx"
      },
     {},
     {} 
   ]
 }
}
```

But in terraform, it seems we are using blocks to represent  a list, so the config will look like this
```
      customized_capacity_metric_specification {
        metric_data_queries {

        }
        metric_data_queries {

        }
        metric_data_queries {

        }
      }
```

I am not sure in this case whether I should name it `metric_data_queries` vs `metric_data_query` ? Any thought here ?

Reply:
We should follow the naming of the API so it will still be `metric_data_queries`. This one doesn't require change. The comment I left was for a documentation grammar change.